src/OrchardCore.Modules/OrchardCore.ContentFields/Views/ContentPickerFieldSettings.Edit.cshtml

```diff
@@ -30,21 +17,44 @@
     </div>
 </div>
 
-<div id="contentTypesVue" class="mb-3">
-    <label asp-for="DisplayedContentTypes">@T["Content Types"]</label>
-    <span class="hint">@T["The content types to display. Choose at least one."]</span>
-
-    <div class="form-check">
-        <input asp-for="DisplayAllContentTypes" type="checkbox" class="form-check-input" v-model="displayAllContentTypes">
-        <label class="form-check-label" asp-for="DisplayAllContentTypes">@T["Display All Content Types"]</label>
-        <span class="hint dashed">@T["Whether to allow picker to display all content types."]</span>
+<div class="mb-3">
+    <label asp-for="Source">@T["Contained Content Types"]</label>
+    <div class="w-sm-50" id="@Html.IdFor(x => x.Source)">
+        <div class="form-check">
+            <input class="form-check-input" type="radio" asp-for="Source" id="SourceAll" value="@ContentPickerFieldSettingsDriver.AllTypes" checked="@(ContentPickerFieldSettingsDriver.AllTypes.Equals(Model.Source, StringComparison.OrdinalIgnoreCase))">
+            <label for="SourceAll">
+                @T["Display All Content Types"]
+            </label>
+        </div>
+        <div class="form-check">
+            <input class="form-check-input" type="radio" asp-for="Source" id="SourceContentTypes" value="@ContentPickerFieldSettingsDriver.ContentTypes" checked="@(ContentPickerFieldSettingsDriver.ContentTypes.Equals(Model.Source, StringComparison.OrdinalIgnoreCase))">
+            <label for="SourceContentTypes">
+                @T["Content Types"]
+            </label>
+        </div>
+        <div class="form-check">
+            <input class="form-check-input" type="radio" asp-for="Source" id="SourceStereotype" value="@ContentPickerFieldSettingsDriver.Stereotypes" checked="@(ContentPickerFieldSettingsDriver.Stereotypes.Equals(Model.Source, StringComparison.OrdinalIgnoreCase))">
+            <label for="SourceStereotype">
+                @T["Stereotype"]
```

Comment:
something more explicit: "All content types of these stereotypes"

Reply:
Done